e2-studio-star-rx72n-firmware/libs/rx_comm_manager/src/rx_comm_manager.c

```
@@ -750,6 +810,131 @@ static rx_err_t internal_poll_spi(rx_comm_manager_t* mgr)
+{
+  if (mgr == nullptr || mgr->event_queue_count == 0) {
+    return;
+  }
+
+  rx_comm_event_entry_t* entry = &mgr->event_queue[mgr->event_queue_tail];
+  if (!entry->occupied) {
+    return;
+  }
+
+  /* Build send params from queued entry */
+  const rx_comm_send_params_t params = {
```

Clang (14.0.6)

[warning] variable 'params' is not initialized

(cppcoreguidelines-init-variables)